src/wazuh_db/wdb.h

```diff
@@ -827,9 +832,31 @@ int wdb_remove_database(const char * agent_id);
  *
  * @param [in] stmt The SQL statement to be executed.
  * @param [out] status The status code of the statement execution. If NULL no value is written.
+ * @param [in] column_mode It could be STMT_SINGLE_COLUMN if the query returns only one column,
+ *                         or STMT_MULTI_COLUMN if the query returns more than one column.
+ * @return JSON array with the statement execution results. NULL On error.
+ */
+cJSON* wdb_exec_row_stmt(sqlite3_stmt * stmt, int* status, bool column_mode);
+
+/**
+ * @brief Function to execute one row of an SQL statement and save the result in a single JSON array without column name like:
+ *        ["column_value_1","column_value_2", ...]. The query should return only one column in every step.
+ *
+ * @param [in] stmt The SQL statement to be executed.
+ * @param [out] status The status code of the statement execution. If NULL no value is written.
+ * @return JSON array with the statement execution results. NULL On error.
```

Comment:
Minor typo, On to on.

Reply:
It may have been done like that on purpose, considering that it is the first word in the sentence after a full capital word like **NULL**. Nevertheless, I've added a comma to remove this problem.